akka-serialization-jackson/src/main/resources/reference.conf

```diff
@@ -140,6 +132,18 @@ akka.serialization.jackson {
   # from `serialization-bindings`, but should still be possible to deserialize.
   whitelist-class-prefix = []
 
+  # settings for compression of the payload
+  compression {
+    # Compression algorithm.
+    # - off  : no compression
+    # - gzip : using common java gzip
+    algorithm = off
```

Comment:
The motivation to disable by default is that for CBOR I think it's better to keep it plain cbor binary. It hasn't great space efficiency but probably good enough tradeoff between size and speed. Some numbers in https://github.com/akka/akka/pull/27067